src/ssl.c

```diff
@@ -37431,27 +37436,29 @@ int wolfSSL_EC_KEY_check_key(const WOLFSSL_EC_KEY *key)
 }
 /* End EC_KEY */
 
+/* Calculate and return maximum size of the ECDSA signature for the curve */
 int wolfSSL_ECDSA_size(const WOLFSSL_EC_KEY *key)
 {
     const EC_GROUP *group;
     int bits, bytes;
-    word32 headerSz = 4;   /* 2*ASN_TAG + 2*LEN(ENUM) */
+    word32 headerSz = SIG_HEADER_SZ; /* 2*ASN_TAG + 2*LEN(ENUM) */
```

Comment:
Does the change from 4 to 7 (looks like `SIG_HEADER_SZ` is 7) mean this comment needs to change, too?

Reply:
Yes I did this on purpose. There was a trailing +2 on the previous return value and this includes that the original code was off by one potentially in some cases